themes/ddbasic/sass/components/form/form-class.scss

```diff
@@ -0,0 +1,75 @@
+@import '../../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?